GTMDefines.h

```diff
@@ -130,11 +130,16 @@
 #if !defined(NS_BLOCK_ASSERTIONS)
   #define _GTMDevAssert(condition, ...)                                       \
     do {                                                                      \
-      if (!(condition)) {                                                     \
+      if (__builtin_expect(!(condition), 0)) {                                \
+        NSString *__assert_func_name__ =                                      \
+            [NSString stringWithUTF8String:__PRETTY_FUNCTION__];              \
+        __assert_func_name__ = __assert_func_name__ ?: @"<Unknown Function>"; \
```

Comment:
Should this just get inlined into the handleFailureInFunction to avoid the extra assignment here?

Reply:
If there's a strong preference to do so I can, but macros are hard enough to read without the line-break contortions that would probably introduce to conform with the Google style guide.

Reply:
@dmaclach any preference here?

Reply:
I'm happy with what mwyman has here.